code/src/urls/urls.js

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line camelcase
```

Comment:
nice, this might be a good thing to take with me ahead, didn't think of it but it does tidy up a bit when handling multiple API's!